### ISSUE TITLE
Wrong Translation string in the frontend

### ISSUE THREAD

**Issue by jonaelsasser** [panelalpha/PanelAlpha-Translations#11]
The modal is referring to the wrong strings upon activating Plugins:
![image](https://github.com/user-attachments/assets/608e6201-8110-41ea-b5f5-d2ae8ccd732f)

The ones I suspect that should be used are these:

```
"plugin.activate.info_one": "Are you sure that you want to activate this plugin?",
"plugin.activate.info_other": "Are you sure that you want to activate the selected plugins?",
```

They are already available in every language.

**Comment by damianwiluszeu** [comment]
Thank you for reporting this problem, we will correct this issue in an upcoming update.

**Comment by jonaelsasser** [comment]
Found another one:

on instances/id/general while hovering over the website thumbnail
![image](https://github.com/user-attachments/assets/6a537385-9657-4027-afd0-c545c7f3fb04)

adding `"refresh.thumbnail": "Thumbnail aktualisieren",` withing `"button": {}` in the translation didn't fix it, so there must be a typo in the frontend aswell.

@damianwiluszeu shall I add the translation string to every language? (so it's working as soon as the frontend part is fixed)

**Comment by damianwiluszeu** [comment]
> The ones I suspect that should be used are these:
```
"plugin.activate.info_one": "Are you sure that you want to activate this plugin?",
"plugin.activate.info_other": "Are you sure that you want to activate the selected plugins?",
```
We have implemented a fix for this in version v1.1.4, now it displays langs correctly. 


> adding "refresh.thumbnail": "Thumbnail aktualisieren", withing "button": {} in the translation didn't fix it, so there must be a typo in the frontend aswell.

As far as I can see this is correctly implemented in Client Area: 
![image](https://github.com/user-attachments/assets/7984b3f8-4372-414e-8337-b397b67ea248)

My tests also show that after adding the line `"refresh.thumbnail: "Thumbnail aktualisieren"` under `"button":` it displays correctly in the Client Area, could you confirm that this problem still occurs with you?

![obraz](https://github.com/user-attachments/assets/26cb09dd-858c-433c-9186-4d67312f0b9e)


> @damianwiluszeu shall I add the translation string to every language? (so it's working as soon as the frontend part is fixed)

That would be really helpful!

**Comment by jonaelsasser** [comment]
uh sorry for the late response, I didn't notice your comment until now.

All the issues above are fixed in german indeed.
